docs/Migrating.md

```diff
@@ -42,6 +27,25 @@ using the search bar of the `Hierarchy` window with the word `Agent`.
   [Inference Engine](Inference-Engine.md).
 * To use a `.tf` model, drag it inside the `Model` property of the `Learning Brain`
 
+#### Steps to Migrate
+
+* To update a scene from v0.5 to v0.6, you must:
+  * Remove the `Brain` GameObjects in the scene. (Delete all of the
+    Brain GameObjects under Academy in the scene.)
+  * Create new `Brain` Scriptable Objects using `Assets -> Create ->
+    ML-Agents` for each type of the Brain you plan to use, and put
+    the created files under a folder called Brains within your project.
+  * Edit their `Brain Parameters` to be the same as the parameters used
+    in the `Brain` GameObjects.
+  * Agents have a `Brain` field in the Inspector, you need to drag the
+    appropriate Brain ScriptableObject in it.
+  * Academies have a `Broadcast Hub` field in the inspector, which is
```

Comment:
Replace `Academies have` with `Academy has`. The broadcast hub does not need to have all of the brains present in the scene. Only the ones that need to be broadcasted or controlled by the Python process.